guide/installation/k8s_install/k8s_install_kubespray.md

```diff
@@ -80,7 +80,27 @@ kube_node
 calico_rr
 ```
 
-## Step 3. Deploy Kubernetes
+## Step 3. Define Kubernetes configuration
+
+Kubespray gives you ability to customize Kubernetes instalation, for example define:
+- network plugin
+- container manager
+- kube_apiserver_port
+- kube_pods_subnet
+- all K&s addons configurations, or even define to deploy cluster on hyperscaller like AWS or GCP.
```

Comment:
I am not Kubernetes expertise, but the "K&s" looks like a typo for me here. you meant k8s? the line 93 and 95 have same issues here.

Reply:
DCO -- https://github.com/apps/dco
Basically, in your comment messages need a -s with your email matching up with your github ID.  Then when you go git log in a repo you get information about you made the commit. eg:  Signed-off-by: letonghan <letong.han@intel.com> 
"git commit -a -s <email> -m "something"
You could also configure this in your .gitconfig